src/components/__tests__/VesselsListItem.spec.js

```diff
@@ -6,36 +6,48 @@ import VesselsListItemActions from '@/components/VesselsListItemActions.vue'
 import { firstVessel } from '@/../test/stubs/vessel'
 
 describe('VesselsListItem.vue', () => {
-  const setup = () => {
+  const deleteVesselTaskSpy = (isActive, isResolved) => {
```

Comment:
Looks like this `TaskSpy` will always have a same properties and shape. Maybe we could consider extracting it to another file?